src/stock-operations/add-stock-operation/add-stock-operation.component.tsx

```diff
@@ -142,347 +161,364 @@ const AddStockOperation: React.FC<AddStockOperationProps> = (props) => {
 
   return (
     <>
-      <div
-        style={{
-          display: "flex",
-          justifyContent: "space-between",
-          margin: "5px",
-        }}
-      >
-        <div style={{ margin: "10px" }}>
-          {isEditing && (
-            <div style={{ display: "flex", flexDirection: "row" }}>
-              <span>Status:</span>
-              <span
-                style={{
-                  marginLeft: "2px",
-                  color: `${operationStatusColor(props?.model?.status)}`,
-                }}
-              >
-                {props?.model?.status}
-              </span>
-            </div>
-          )}
-          <div
-            style={{ display: "flex", flexDirection: "row", marginTop: "4px" }}
-          >
-            {props?.model?.dateCreated && (
-              <div>
-                <span>Started: </span>
-                <div
+      {!isEditing && props.operation.name === "Stock Issue" ? (
+        <></>
+      ) : (
+        <div
+          style={{
+            display: "flex",
+            justifyContent: "space-between",
+            margin: "5px",
```

Comment:
@ It will be good if we have this in the `.scss` file, and will make managing of css very easy.

Reply:
@makombe let me look into this